lib/archethic_web/templates/explorer/reward_chain_index.html.leex

```diff
@@ -0,0 +1,56 @@
+<article class="message is-info">
+  <div class="message-header ">
+   <h1 class="subtitle is-size-4 heading has-text-white">Reward Chain</h2>
+  </div>
+  <div class="message-body">
+  Reward Chains are the basis for remuneration. During self-repair,  <strong>Archethic Rewards</strong> are created and distributed to Miners.
```

Comment:
This is not really accurate, actually the rewards are not sent during self-repair, but the fees burnt are collected during self-repair. The rewards transactions are sent at a specific interval later, once per cycle.